packages/react-native-gesture-handler/src/v3/hooks/gestures/fling/useFling.ts

```diff
@@ -39,9 +39,11 @@ export type FlingGesture = SingleGesture<
 >;
 
 export function useFling(config: FlingGestureConfig): FlingGesture {
-  const flingConfig = cloneConfig<FlingHandlerData, FlingGestureProperties>(
-    config
-  );
+  const flingConfig = useClonedAndRemappedConfig<
+    FlingHandlerData,
+    FlingGestureProperties,
+    FlingGestureProperties
```

Comment:
If last generic parameter is optional, do we need to pass it? I'd choose either making it required, or do not pass it when not necessary.